### PR TITLE
CLDR-16135 Add more sample names

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-16135

- [x] This PR completes the ticket.

### CODE REVIEW

File: common/main/az.xml [hunk 1]
```diff
@@ -10311,27 +10311,27 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
...
-		<sampleName item="given12Surname">
+		<sampleName item="nativeGGS">
 			<nameField type="given" draft="contributed">Con</nameField>
 			<nameField type="given2" draft="contributed">Hamiş</nameField>
 			<nameField type="surname" draft="contributed">Vatson</nameField>
 		</sampleName>
-		<sampleName item="full">
+		<sampleName item="foreignFull">
+			<nameField type="surname-prefix">van den</nameField>
+			<nameField type="suffix">M.D. Ph.D.</nameField>
```
Comment (pedberg-icu) [inline common/main/az.xml:10328]: Not sure why these got re-ordered, but CLDRModify will eventualy fix.

**Comment by pedberg-icu** [comment]
Hmm, github will not show me any of the changes past common/main/da.xml, it just says "Oops, something went wrong" with a link to Retry; clicking the link just ends up with the same result. I tried in Safari and Firefox,and also (on a personal machine) in Chrome.

**Comment by macchiati** [comment]
Dang, I meant to have those exemplar changes in a different PR. Will fix.

**Comment by pedberg-icu** [comment]
> Dang, I meant to have those exemplar changes in a different PR. Will fix.

Can you actually see the file changes (past da.xml, such as the code changes) in github?

**Comment by macchiati** [comment]
I put in a fix for the exemplars (and also the dtd).

You can see the extra files if you use magic. Hit a period (when not editing anything) and it opens up https://github.dev/unicode-org/cldr/pull/2547/files

Steven clued me onto this.

**Comment by pedberg-icu** [comment]
There are 5 errors in ConsoleCheck:
```
2022-11-16T01:20:45.9262639Z /home/runner/work/cldr/cldr/common/main/fa.xml:11324:70: error
2022-11-16T01:20:45.9367681Z fa [Persian]	error	▸Miscellaneous|Person_Name_Formats|SampleName_Fields_for_Item:_foreignFull|suffix◂	〈MD DDS〉	【】	〈دکترای پزشکی پی\x{200C}اچ.دی.〉	«=»	【】	⁅internal error⁆	❮Error: Internal error in org.unicode.cldr.test.CheckForExemplars. Exception: java.lang.NumberFormatException, Message: java.lang.NumberFormatException: For input string: "200C", Trace: [java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65), java.base/java.lang.Integer.parseInt(Integer.java:652), org.unicode.cldr.test.CheckForExemplars.checkAndReplacePlaceholders(CheckForExemplars.java:528), org.unicode.cldr.test.CheckForExemplars.handleCheck(CheckForExemplars.java:307), org.unicode.cldr.test.CheckCLDR$CompoundCheckCLDR.handleCheck(CheckCLDR.java:1224), org.unicode.cldr.test.CheckCLDR.check(CheckCLDR.java:1121), org.unicode.cldr.test.ConsoleCheckCLDR.main(ConsoleCheckCLDR.java:626), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:566), org.unicode.cldr.tool.Main.invoke(Main.java:59), org.unicode.cldr.tool.Main.main(Main.java:49)]; For input string: "200C"❯	https://st.unicode.org/cldr-apps/v#/fa//1e89d97cd974975b
2022-11-16T01:20:45.9518288Z ##[error] Error: Internal error in org.unicode.cldr.test.CheckForExemplars. Exception: java.lang.NumberFormatException, Message: java.lang.NumberFormatException: For input string: "200C", Trace: [java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65), java.base/java.lang.Integer.parseInt(Integer.java:652), org.unicode.cldr.test.CheckForExemplars.checkAndReplacePlaceholders(CheckForExemplars.java:528), org.unicode.cldr.test.CheckForExemplars.handleCheck(CheckForExemplars.java:307), org.unicode.cldr.test.CheckCLDR$CompoundCheckCLDR.handleCheck(CheckCLDR.java:1224), org.unicode.cldr.test.CheckCLDR.check(CheckCLDR.java:1121), org.unicode.cldr.test.ConsoleCheckCLDR.main(ConsoleCheckCLDR.java:626), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:566), org.unicode.cldr.tool.Main.invoke(Main.java:59), org.unicode.cldr.tool.Main.main(Main.java:49)]; For input string: "200C"
2022-11-16T01:20:45.9529262Z # fa [Persian]	Summary	modern	Subtotal error:	1

2022-11-16T01:22:26.7070720Z /home/runner/work/cldr/cldr/common/main/kn.xml:12306:56: error
2022-11-16T01:22:26.7074683Z kn [Kannada]	error	▸Miscellaneous|Person_Name_Formats|SampleName_Fields_for_Item:_nativeG|given◂	〈Zendaya〉	【】	〈ಸಿನ್\x{200C}ಬಾದ್〉	«=»	【】	⁅internal error⁆	❮Error: Internal error in org.unicode.cldr.test.CheckForExemplars. Exception: java.lang.NumberFormatException, Message: java.lang.NumberFormatException: For input string: "200C", Trace: [java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65), java.base/java.lang.Integer.parseInt(Integer.java:652), org.unicode.cldr.test.CheckForExemplars.checkAndReplacePlaceholders(CheckForExemplars.java:528), org.unicode.cldr.test.CheckForExemplars.handleCheck(CheckForExemplars.java:307), org.unicode.cldr.test.CheckCLDR$CompoundCheckCLDR.handleCheck(CheckCLDR.java:1224), org.unicode.cldr.test.CheckCLDR.check(CheckCLDR.java:1121), org.unicode.cldr.test.ConsoleCheckCLDR.main(ConsoleCheckCLDR.java:626), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:566), org.unicode.cldr.tool.Main.invoke(Main.java:59), org.unicode.cldr.tool.Main.main(Main.java:49)]; For input string: "200C"❯	https://st.unicode.org/cldr-apps/v#/kn//344f8c92944438d7
2022-11-16T01:22:26.7084217Z ##[error] Error: Internal error in org.unicode.cldr.test.CheckForExemplars. Exception: java.lang.NumberFormatException, Message: java.lang.NumberFormatException: For input string: "200C", Trace: [java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65), java.base/java.lang.Integer.parseInt(Integer.java:652), org.unicode.cldr.test.CheckForExemplars.checkAndReplacePlaceholders(CheckForExemplars.java:528), org.unicode.cldr.test.CheckForExemplars.handleCheck(CheckForExemplars.java:307), org.unicode.cldr.test.CheckCLDR$CompoundCheckCLDR.handleCheck(CheckCLDR.java:1224), org.unicode.cldr.test.CheckCLDR.check(CheckCLDR.java:1121), org.unicode.cldr.test.ConsoleCheckCLDR.main(ConsoleCheckCLDR.java:626), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:566), org.unicode.cldr.tool.Main.invoke(Main.java:59), org.unicode.cldr.tool.Main.main(Main.java:49)]; For input string: "200C"
2022-11-16T01:22:26.7094990Z # kn [Kannada]	Summary	modern	Subtotal error:	1

2022-11-16T01:23:03.1691891Z /home/runner/work/cldr/cldr/common/main/ml.xml:11845:55: error
2022-11-16T01:23:03.1696153Z ml [Malayalam]	error	▸Miscellaneous|Person_Name_Formats|SampleName_Fields_for_Item:_nativeGS|surname◂	〈Adler〉	【】	〈ആഡ്\x{200C}ലർ〉	«=»	【】	⁅internal error⁆	❮Error: Internal error in org.unicode.cldr.test.CheckForExemplars. Exception: java.lang.NumberFormatException, Message: java.lang.NumberFormatException: For input string: "200C", Trace: [java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65), java.base/java.lang.Integer.parseInt(Integer.java:652), org.unicode.cldr.test.CheckForExemplars.checkAndReplacePlaceholders(CheckForExemplars.java:528), org.unicode.cldr.test.CheckForExemplars.handleCheck(CheckForExemplars.java:307), org.unicode.cldr.test.CheckCLDR$CompoundCheckCLDR.handleCheck(CheckCLDR.java:1224), org.unicode.cldr.test.CheckCLDR.check(CheckCLDR.java:1121), org.unicode.cldr.test.ConsoleCheckCLDR.main(ConsoleCheckCLDR.java:626), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:566), org.unicode.cldr.tool.Main.invoke(Main.java:59), org.unicode.cldr.tool.Main.main(Main.java:49)]; For input string: "200C"❯	https://st.unicode.org/cldr-apps/v#/ml//7dc36fe92cd5f2c4
2022-11-16T01:23:03.1703625Z ##[error] Error: Internal error in org.unicode.cldr.test.CheckForExemplars. Exception: java.lang.NumberFormatException, Message: java.lang.NumberFormatException: For input string: "200C", Trace: [java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65), java.base/java.lang.Integer.parseInt(Integer.java:652), org.unicode.cldr.test.CheckForExemplars.checkAndReplacePlaceholders(CheckForExemplars.java:528), org.unicode.cldr.test.CheckForExemplars.handleCheck(CheckForExemplars.java:307), org.unicode.cldr.test.CheckCLDR$CompoundCheckCLDR.handleCheck(CheckCLDR.java:1224), org.unicode.cldr.test.CheckCLDR.check(CheckCLDR.java:1121), org.unicode.cldr.test.ConsoleCheckCLDR.main(ConsoleCheckCLDR.java:626), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:566), org.unicode.cldr.tool.Main.invoke(Main.java:59), org.unicode.cldr.tool.Main.main(Main.java:49)]; For input string: "200C"
2022-11-16T01:23:03.1711106Z /home/runner/work/cldr/cldr/common/main/ml.xml:11854:61: error
2022-11-16T01:23:03.1714702Z ml [Malayalam]	error	▸Miscellaneous|Person_Name_Formats|SampleName_Fields_for_Item:_foreignFull|given◂	〈Ada Cornelia〉	【】	〈എയ്\x{200C}ഡ കോർണീലിയ〉	«=»	【】	⁅internal error⁆	❮Error: Internal error in org.unicode.cldr.test.CheckForExemplars. Exception: java.lang.NumberFormatException, Message: java.lang.NumberFormatException: For input string: "200C", Trace: [java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65), java.base/java.lang.Integer.parseInt(Integer.java:652), org.unicode.cldr.test.CheckForExemplars.checkAndReplacePlaceholders(CheckForExemplars.java:528), org.unicode.cldr.test.CheckForExemplars.handleCheck(CheckForExemplars.java:307), org.unicode.cldr.test.CheckCLDR$CompoundCheckCLDR.handleCheck(CheckCLDR.java:1224), org.unicode.cldr.test.CheckCLDR.check(CheckCLDR.java:1121), org.unicode.cldr.test.ConsoleCheckCLDR.main(ConsoleCheckCLDR.java:626), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:566), org.unicode.cldr.tool.Main.invoke(Main.java:59), org.unicode.cldr.tool.Main.main(Main.java:49)]; For input string: "200C"❯	https://st.unicode.org/cldr-apps/v#/ml//50e3fa63ae101746
2022-11-16T01:23:03.1721433Z ##[error] Error: Internal error in org.unicode.cldr.test.CheckForExemplars. Exception: java.lang.NumberFormatException, Message: java.lang.NumberFormatException: For input string: "200C", Trace: [java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65), java.base/java.lang.Integer.parseInt(Integer.java:652), org.unicode.cldr.test.CheckForExemplars.checkAndReplacePlaceholders(CheckForExemplars.java:528), org.unicode.cldr.test.CheckForExemplars.handleCheck(CheckForExemplars.java:307), org.unicode.cldr.test.CheckCLDR$CompoundCheckCLDR.handleCheck(CheckCLDR.java:1224), org.unicode.cldr.test.CheckCLDR.check(CheckCLDR.java:1121), org.unicode.cldr.test.ConsoleCheckCLDR.main(ConsoleCheckCLDR.java:626), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:566), org.unicode.cldr.tool.Main.invoke(Main.java:59), org.unicode.cldr.tool.Main.main(Main.java:49)]; For input string: "200C"
2022-11-16T01:23:03.1727457Z # ml [Malayalam]	Summary	modern	Subtotal error:	2

2022-11-16T01:23:28.4505604Z /home/runner/work/cldr/cldr/common/main/nl_BE.xml:215:56: error
2022-11-16T01:23:28.4506728Z nl_BE [Flemish]	error	▸Miscellaneous|Person_Name_Formats|SampleName_Fields_for_Item:_nativeFull|surname-prefix◂	〈∅∅∅〉	【】	〈Van den〉	«=»	【】	⁅invalid place holder⁆	❮Error: cannot have -prefix without -core❯	https://st.unicode.org/cldr-apps/v#/nl_BE//2954d138834686ee
2022-11-16T01:23:28.4508487Z ##[error] Error: cannot have -prefix without -core
2022-11-16T01:23:28.4509965Z # nl_BE [Flemish]	Summary	modern	Subtotal error:	1
```

File: common/main/en.xml [hunk 2]
```diff
@@ -9431,27 +9431,49 @@ annotations.
...
-			<nameField type="surname-prefix">van den</nameField>
+			<nameField type="surname-prefix">∅∅∅</nameField>
 			<nameField type="surname-core">Wolf</nameField>
-			<nameField type="surname2">Becker Schmidt</nameField>
+			<nameField type="surname2">∅∅∅</nameField>
 			<nameField type="suffix">M.D. Ph.D.</nameField>
 		</sampleName>
+		<sampleName item="foreignG">
+			<nameField type="given">Zendaya</nameField>
+		</sampleName>
```
Comment (pedberg-icu) [inline common/main/en.xml:9458]: Will vetters be confused by having the foreig names in English be (mostly) the same as native names?

File: common/main/nl_BE.xml [hunk 3]
```diff
@@ -199,25 +199,25 @@ CLDR data files are interpreted according to the LDML specification (http://unic
...
 		</sampleName>
-		<sampleName item="given12Surname">
+		<sampleName item="nativeGGS">
 			<nameField type="given" draft="provisional">Liam</nameField>
 			<nameField type="given2" draft="provisional">Hugo Mees</nameField>
 			<nameField type="surname" draft="provisional">Van den Berg</nameField>
 		</sampleName>
-		<sampleName item="full">
+		<sampleName item="nativeFull">
+			<nameField type="surname-prefix">Van den</nameField>
```
Comment (pedberg-icu) [inline common/main/nl_BE.xml:215]: Another funny reordering, CLDRModify should fix later

File: tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckPersonNames.java [hunk 4]
```diff
@@ -17,64 +18,116 @@ public class CheckPersonNames extends CheckCLDR {
      * @internal, public for testing
      */
     public static final ImmutableMultimap<SampleType, String> REQUIRED = ImmutableMultimap.<SampleType, String> builder()
-        .putAll(SampleType.givenOnly, "given")
-        .putAll(SampleType.givenSurnameOnly, "given", "surname")
-        .putAll(SampleType.given12Surname, "given", "given2", "surname")
-        .putAll(SampleType.full, "prefix", "given", "given-informal", "given2", "surname", "surname2", "suffix")
+        .putAll(SampleType.nativeG, "given")
```
Comment (pedberg-icu) [inline tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckPersonNames.java:21]: THis and the next 3 duplicate or conflict with later entries here

File: tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckPersonNames.java [hunk 5]
```diff
@@ -17,64 +18,116 @@ public class CheckPersonNames extends CheckCLDR {
...
+        .putAll(SampleType.foreignGGS, "given", "given2", "surname")
+        .putAll(SampleType.foreignFull, "prefix", "given", "given-informal", "given2", "surname", "surname2", "suffix")
         .build();
     /**
      * @internal, public for testing
      */
     public static final ImmutableMultimap<SampleType, String> REQUIRED_EMPTY = ImmutableMultimap.<SampleType, String> builder()
-        .putAll(SampleType.givenOnly, "prefix", "given-informal", "given2", "surname", "surname-prefix", "surname-core", "surname2", "suffix")
-        .putAll(SampleType.givenSurnameOnly, "prefix", "given2", "surname-prefix", "surname-core", "surname2", "suffix")
+        .putAll(SampleType.nativeG, "prefix", "given-informal", "given2", "surname", "surname-prefix", "surname-core", "surname2", "suffix")
```
Comment (pedberg-icu) [inline tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckPersonNames.java:38]: Again, aren't we duplicating some entries here?

**Review by pedberg-icu — COMMENTED** [review]
This looks pretty good, I just had a few comments/questions, and there are some test failures. Will check again soon.

**Review by pedberg-icu — APPROVED** [review]
I am going to go ahead and approve, if any changes are needed per my questions we can fix in a later PR

**Comment by macchiati** [comment]
I think I got in fixes for all of your comments (except the ordering of elements, where we both agree those will automatically be fixed).

**Comment by macchiati** [comment]
Not quite all. Will have separate PR for the en.xml changes and the CheckPersonNames changes.

**Pull request merged**